src/test/scala/chiselTests/FixedIOModuleSpec.scala

```diff
@@ -88,4 +92,165 @@ class FixedIOModuleSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
       "input end : UInt<1>"
     )()
   }
+
+  class Agg extends Bundle {
+    val foo = Bool()
+    val bar = Bool()
+  }
+
+  class Nested extends Bundle {
+    val foo = Probe(Bool())
+    val bar = Probe(Bool())
+  }
+
+  class FixedIO(probeAgg: Boolean, aggProbe: Boolean) extends Bundle {
```

Comment:
This is very nitty, but these names are a little weird, `Nested` isn't really `Nested` and `FixedIO` is just a Bundle, it's use in `FixedIORawModule` is a separate thing. I'd call `FixedIO` something like `MixedProbe`

Reply:
changed to AggWithProbes and MixedBundle